server.lua

```diff
@@ -1,7 +1,20 @@
 local Zones = nil
+ESX = nil
+
+TriggerEvent('esx:getSharedObject', function(obj) ESX = obj end)
 
 function isPlayerAuthorized(source)
-    if (Config.USE_ESSENTIALMODE_ADMIN_SYSTEM) then
+    if (Config.USE_ESX_ADMIN_SYSTEM) then
+        -- Fetch the user and check permissions
+        local xPlayer = ESX.GetPlayerFromId(source)
+        local plyGroup = xPlayer.getGroup()
```

Comment:
Hmm, sometimes that's returns `SCRIPT ERROR: @iZone/server.lua:10: attempt to index a nil value (local 'xPlayer')`. I'll check it